R/bootstrap-layout.R

```diff
@@ -346,6 +353,8 @@ sidebarLayout <- function(sidebarPanel,
 #' @param fluid `TRUE` to use fluid layout; `FALSE` to use fixed
 #'   layout.
 #'
+#' @family layout functions
+#'
 #' @seealso [fluidPage()], [flowLayout()]
```

Comment:
This `@seealso` section should be removed since `fluidPage()` and `flowLayout()` are part of the layout family